src/main/java/cceclipseplugin/editor/DocumentManager.java

```diff
@@ -73,7 +73,7 @@ public void setCurrFile(String filePath) {
 			this.currFile = null;
 			return;
 		}
-		this.currFile = Paths.get(filePath).toString();
+		this.currFile = filePath;
```

Comment:
Probably should update Java docs to more explicitly denote what kind of paths should be used in this instance (absolute paths)